docs/source/models/cause.rst

```diff
@@ -558,6 +558,113 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+**Definition**
+
+Remission is a measure of cases that recover from a with-condition state, given 
+a specified population and time period. Just as with incidence, the numerator is 
+given by the count of recovered (or *remitted*) cases, and the denominator is 
+the cumulative person-time [link to person-time definition in incidence 
+section] during which cases are able to go into remission.
+
+  For example, consider diarrhea cases in the Philippines in 2017. Say there
+  were 53,362,490 prevalent cases of diarrhea over the course of 2017, and each
+  case remitted after an average of 5 days. Note that here we are using *period
+  prevalence*; however, GBD (and thus we) typically mean *point prevalence* when 
+  we refer to prevalence. Then, the way we then define this remission rate 
+  depends on the denominator we choose. Some examples include:
+
+  .. math::
+
+    \frac{\text{1/5 case}}{\text{1 person-days}}=\frac{365/5\text{ cases}}{\text{1 person-year}}=73\text{ cases/person-year}
+
+  We also note that in the context of a one-year timestep, as diarrheal diseases
+  have such a high remission rate relative to one year, period prevalence is
+  effectively the same as incidence. We can see how this would not be the case
```

Comment:
This is not immediately obvious to me. Perhaps it will become obvious if I spend a minute or two thinking about it, but it's probably better to add a brief explanation of why period prevalence is close to incidence in this case so I don't have to spend that minute or two...